packages/tools/devtools/devtools-view/src/components/OpLatencyView.tsx

```diff
@@ -44,6 +51,109 @@ const useStyles = makeStyles({
 export function OpLatencyView(): React.ReactElement {
 	const styles = useStyles();
 
+	const messageRelay = useMessageRelay();
+	const [durationOutboundBatchingData, setDurationOutboundBatchingData] =
+		React.useState<GraphDataSet>({
+			graphType: "line",
+			schema: {
+				displayName: "Duration Outbound",
+				uuid: "durationOutbound",
+				xAxisDataKey: "timestamp",
+				yAxisDataKey: "duration",
+			},
+			data: [],
+		});
+	const [durationNetworkData, setDurationNetworkData] = React.useState<GraphDataSet>({
+		graphType: "line",
+		schema: {
+			displayName: "Duration Network",
+			uuid: "durationNetwork",
+			xAxisDataKey: "timestamp",
+			yAxisDataKey: "duration",
+		},
+		data: [],
+	});
+	const [durationInboundToProcessingData, setDurationInboundToProcessingData] =
+		React.useState<GraphDataSet>({
+			graphType: "line",
+			schema: {
+				displayName: "Duration Inbound",
+				uuid: "durationInbound",
+				xAxisDataKey: "timestamp",
+				yAxisDataKey: "duration",
+			},
+			data: [],
+		});
+	const unsampledTelemetry = localStorage.getItem("Fluid.Telemetry.DisableSampling");
+	// Render the text conditionally
+	const renderText = unsampledTelemetry !== "true";
+
+	React.useEffect(() => {
+		/**
+		 * Handlers for inbound messages.
+		 */
+		const inboundMessageHandlers: InboundHandlers = {
+			[TelemetryEvent.MessageType]: async (untypedMessage) => {
+				const message = untypedMessage as TelemetryEvent.Message;
+				const eventContents = message.data.event.logContent;
+				// We ignore the ops related to round trip time or summarizer as Op latency is focused on ops generated by the user
```

Comment:
The first part is actually the other way around. Also see my other comment about the summarizer
```suggestion
				// We skip telemetry events that are not related to op roundtrip time and ignore events from the summarizer,
				// since Op latency is focused on ops generated by the user
```